src/nyc_rest_eda.py

```diff
@@ -0,0 +1,216 @@
+# author: Lauren Zung
+# date: 2022-11-24
+
+"""Creates new tables and plots of the training data from the DOHMH New York City Restaurant Inspection Results as part of the project's exploratory data analysis.
+Saves the tables and plots as png files.
+
+Original data source: https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2018/2018-12-11/nyc_restaurants.csv
+
+Usage: src/nyc_rest_eda.py --train_set=<train_set> --visual_dir=<visual_dir>
+
+Options:
+--train_set=<train_set>           Path to the training data (in data/preprocessed)
+--visual_dir=<visual_dir>         Path to the output directory for the tables and plots (src/nyc_rest_eda_script_visuals)
+
+Command to run the script:
+python src/nyc_rest_eda.py --train_set='./data/processed/train_df.csv' --visual_dir="src/nyc_rest_eda_script_visuals"
```

Comment:
we could follow the single quote convention for both the parameter values for uniformity